OloEngine/src/OloEngine/Scene/SceneSerializer.cpp

```
@@ -49,6 +50,243 @@ namespace OloEngine
+        TrySet(emitter.InitialColor, particleComponent["InitialColor"]);
+        // Emission shape deserialization
+        if (auto shapeType = particleComponent["EmissionShapeType"]; shapeType)
+        {
+            switch (static_cast<EmissionShapeType>(shapeType.as<int>()))
+            {
+                case EmissionShapeType::Point:
+                    emitter.Shape = EmitPoint{};
+                    break;
+                case EmissionShapeType::Sphere:
+                {
+                    EmitSphere s;
```

Clang (14.0.6)

[warning] variable 's' is not initialized

(cppcoreguidelines-init-variables)